### PR TITLE
MWPW-143951: Optimize sidenav

### PR DESCRIPTION
**Pull request by yesil**
move merch-card-collection and sidenav inside
catalog block.

includes: https://git.corp.adobe.com/wcms/tacocat.js/pull/552/files

depends on: https://github.com/adobecom/milo/pull/2057

Resolves: [MWPW-143951](https://jira.corp.adobe.com/browse/MWPW-143951)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/drafts/ilyas/catalog-sidenav?martech=off&georouting=off
- After: https://mwpw-143951--cc--yesil.hlx.live/drafts/ilyas/catalog?martech=off&georouting=off

### CODE REVIEW

File: creativecloud/scripts/scripts.js [hunk 1]
```diff
@@ -163,6 +162,14 @@ const CONFIG = {
  * ------------------------------------------------------------
  */
 
+const observer = new PerformanceObserver((list) => {
+  list.getEntries().forEach((entry) => {
+    console.log(entry);
```
Comment (aishwaryamathuria) [inline creativecloud/scripts/scripts.js:167]: @yesil Could you please remove the console log.

Comment (yesil) [reply creativecloud/scripts/scripts.js:167]: @aishwaryamathuria good catch, I need to remove the whole `PerformanceObserver`, thanks.

**Review by 3ch023 — APPROVED** [review]

**Comment by salonijain3** [comment]
@yesil can you add the "ready for stage" label if it is ready and get it verified.

**Comment by salonijain3** [comment]
Also, can you add the live urls so that AEM PSI check can pass

**Comment by yesil** [comment]
@salonijain3 this depends on https://github.com/adobecom/milo/pull/2057, and I'll tag ready for stage after it is merged.

**Comment by yesil** [comment]
@salonijain3 this is ready for stage.

**Comment by salonijain3** [comment]
@yesil  we need the "verified" label from QA, so that this can be marked "ready for stage".
cc: @aishwaryamathuria

**Comment by aishwaryamathuria** [comment]
@yesil Seems like the Milo PR was merged. Could you add "Ready for stage" label if this PR is ready to be merged?

**Comment by yesil** [comment]
@aishwaryamathuria  this is ready for stage now, thanks.

**Pull request merged**